tests/test_patroni.py

```diff
@@ -54,6 +54,7 @@ def test_no_config(self):
         self.assertRaises(SystemExit, patroni_main)
 
     @patch('sys.argv', ['patroni.py', '--validate-config', 'postgres0.yml'])
+    @patch('socket.socket.connect_ex', Mock(return_value=1))
```

Comment:
[Off-topic] I am wondering, is validate_host_port() with arg listen=False covered anyhow? 

https://github.com/zalando/patroni/blob/641647eeacbcdfeaba91aa48631d2018d8d06176/patroni/validator.py#L48-L52

Reply:
They are (according to coverage).
And the `connect_ex()` is mocked here: https://github.com/zalando/patroni/blob/db9b5962ec5b921c4c98823f623084f6be58b648/tests/test_validator.py#L111-L131

Reply:
oh, thanks. Missed that part

Reply:
although I still don't see any explicit listen=False setting

Reply:
`False` is a default value.
Calls happen from here: https://github.com/zalando/patroni/blob/db9b5962ec5b921c4c98823f623084f6be58b648/patroni/validator.py#L60-L66

Reply:
🤦‍♀️ sure, thanks again